System.IO.Ports/SerialPort.cs

```diff
@@ -14,31 +13,25 @@ namespace System.IO.Ports
     /// </summary>
     public sealed class SerialPort : IDisposable
     {
-        /// <summary>
-        /// Indicates that no time-out should occur.
-        /// </summary>
-        public const int InfiniteTimeout = -1;
-
-        /// <summary>
-        /// Default new Line, you can change and adjust it in the NewLine property
-        /// </summary>
-        public const string DefaultNewLine = "\r\n";
+        // default new line
+        [System.Diagnostics.DebuggerBrowsable(Diagnostics.DebuggerBrowsableState.Never)]
+        private const string _defaultNewLine = "\r";
```

Comment:
Why removing the default const one?

Reply:
Not removing. Just adjusting to follow .NET (see the PR comment, second bullet).